man/fi_mr.3.md

```diff
@@ -554,18 +558,22 @@ struct fi_mr_dmabuf {
 	int      fd;
 	uint64_t offset;
 	size_t   len;
+	void     *base_addr;
 };
 ```
 The fd is the file descriptor associated with the DMA-buf region.  The
 offset is the offset into the region where the memory registration should
 begin.  And len is the size of the region to register, starting at the
-offset.  The selection of dmabuf over the mr_iov field is controlled by
+offset. The base_addr is the base virtual address of the memory region to
```

Comment:
I would modify this to `base_addr is the base address of the dmabuf region that is mapped to the virtual memory space.` IIUC the base_addr + offset is the base virtual address for the registered memory.